solc/CommandLineInterface.cpp

```diff
@@ -770,14 +849,14 @@ bool CommandLineInterface::processInput()
 		if (successful && m_args.count(g_argFormal))
 			if (!m_compiler->prepareFormalAnalysis())
 				successful = false;
-
-		for (auto const& error: m_compiler->errors())
-			SourceReferenceFormatter::printExceptionInformation(
-				cerr,
-				*error,
-				(error->type() == Error::Type::Warning) ? "Warning" : "Error",
-				scannerFromSourceName
-			);
+		if (!m_compiler->importedSources())
```

Comment:
I think warnings should be kept. Does it crashes or just the source references are wrong?